llvm/include/llvm/IR/IntrinsicsRISCV.td

```diff
@@ -108,6 +108,29 @@ let TargetPrefix = "riscv" in {
   def int_riscv_xperm8  : BitManipGPRGPRIntrinsics;
 } // TargetPrefix = "riscv"
 
+//===----------------------------------------------------------------------===//
+// May-Be-Operations
+
+let TargetPrefix = "riscv" in {
+
+  class MOPGPRIntrinsics
+      : DefaultAttrsIntrinsic<[llvm_any_ty],
+                              [LLVMMatchType<0>],
+                              [IntrNoMem, IntrSpeculatable]>;
```

Comment:
These properties make these intrinsics useless for implementing any of the defined overrides of may-be-operations like SSPUSH or SSPOPCHK. But that's how the intrinsic spec is written so this is fine.